Models/Request/SubmitSightingRequest.cs

```diff
@@ -0,0 +1,29 @@
+using System;
+using System.ComponentModel.DataAnnotations;
+
+namespace whale_spotting.Models.Request
+{
+    public class SubmitSightingRequest
+    {
+        [Required]
+        [StringLength(20)]
+        public string Species { get; set; }
+        [StringLength(10)]
+        public string Quantity { get; set; }
+        [StringLength(20)]
```

Comment:
This  might be a bit short for both location and species, I imagine that some of the longer location names might exceed this, being generous with something like 70 characters is probably still fine as it means they can't type anything dangerous still

Reply:
resolved